services/fission-user-workers/service/dockerworker/main.py

```diff
@@ -0,0 +1,98 @@
+import requests
+import os
+import sys
+import importlib
+from zipfile import ZipFile
+from base64 import b64decode
+import pika
+import flask
+import time
+
+app = flask.Flask("dockerworker")
```

Comment:
not that familiar with Flask, but isnt it meant for more complex web api's? don't know about the impact on resources by using Flask, if it's very lightweight it doesnt really matter however I can image we wan't to keep this script as lightweight and simple as possible

Reply:
Fission uses Flask for their python environments. I tried copying the environment sort of as closely to that in the docker worker. Fortunately the docker worker is for development purposes only, so I don't care too much about speed.